src/south/FolderScanner/FolderScanner.class.js

```diff
@@ -31,35 +31,12 @@ class FolderScanner extends ProtocolHandler {
     this.handlesFiles = true
```

Comment:
At the moment it is redundant, but we can have a South with `supportListen: true` and `supportFile` either `true` or `false`. I think it would be better to separate the support for the source for the South (listen, points, files, history) from the output it provides (points or files)